workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+using Unity.Entities;
+using UnityEngine;
+using Entity = Unity.Entities.Entity;
+
+namespace Improbable.Gdk.Core
+{
+    public class Worker : IDisposable
+    {
+        internal readonly Dictionary<EntityId, Entity> EntityMapping = new Dictionary<EntityId, Entity>();
+
+        public readonly Vector3 Origin;
+        public readonly string WorkerType;
+        public readonly string WorkerId;
+        public readonly World World;
+        public readonly Entity WorkerEntity;
+        public readonly Connection Connection;
+        public readonly ILogDispatcher LogDispatcher;
+
+        public delegate void WorkerEventHandler(Worker worker);
+        public static event WorkerEventHandler OnConnect;
```

Comment:
Having these as static is a bit odd. I'd rather have callbacks or tasks for specific worker than all workers.